internal/graph/dotgraph.go

```diff
@@ -473,13 +474,18 @@ func min64(a, b int64) int64 {
 	return b
 }
 
-// escapeForDot escapes double quotes and backslashes, and replaces Graphviz's
-// "center" character (\n) with a left-justified character.
-// See https://graphviz.org/doc/info/attrs.html#k:escString for more info.
-func escapeForDot(in []string) []string {
+// Applies escapeForDot to all strings in the given slice.
+func escapeAllForDot(in []string) []string {
```

Comment:
You don’t have to comment unexported methods, but when you do, it’s better if you follow the convention: start the comment with the name of thing it describes (https://blog.golang.org/godoc)
So the above code can be improved by changing the comment to:
```
escapeAllForDot applies escapeForDot to all strings in the given slice.
```

Reply:
Fixed in fc32837. Thanks! :)

Also, I marked that commit as `Co-Authored-By: @Segflow`, which seems to have triggered this Google authorship bot.. 😅 Sorry about that. lol

Reply:
Thanks. Done! :)